scripts/salt-extend

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+'''
+Quickstart for creating an/or extending the functionality of your SaltStack installation
+
+    usage: salt-extend [-h] [--extension EXTENSION]
+                   [--salt-directory SALT_DIRECTORY] [--name NAME]
+                   [--description DESCRIPTION] [--no-merge]
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --extension EXTENSION, -e EXTENSION
+                            Extension type, e.g. 'module', 'state'.
+      --salt-directory SALT_DIRECTORY, -o SALT_DIRECTORY
+                            Directory where your salt installation is kept
+                            (defaults to .).
+      --name NAME, -n NAME  Module name.
+      --description DESCRIPTION, -d DESCRIPTION
+                            Short description of what the module does.
+      --no-merge            Don't merge the module into the salt directory, keep
+                            in a temp location
+'''
+import logging
+from salt.scripts import salt_extend
+
+if __name__ == '__main__':
+    import argparse
```

Comment:
`argparse` is not available under python 2.6. Also not a problem since we already use it on other parts of salt(although we do not hard depend on it).

I could ask you to use `optparse` but don't worry about this one, anyone developing salt modules should be doin' so in a recent system, one which at least has python 2.7.